docs/coordinates/performance.inc.rst

```diff
@@ -32,7 +32,7 @@ properties::
 
     >>> from astropy.coordinates import SkyCoord, EarthLocation
     >>> from astropy import coordinates as coord
-    >>> from astropy.coordinates.angle_utilities import golden_spiral_grid
```

Comment:
Hmm, so this was supposedly public... That would mean this changes API and would need deprecation. On the other hand, I think the routine was always available in `astropy.coordinates` itself, so really this is a bug in the documentation and one should just use `coord.golden_spiral_grid` below.

Reply:
We have 2 options: full deprecation period using a module-level ``__getattr__``, or a whatsnew that explains that the top-level import hasn't changed and both old and new code should switch to the top-level import.
Which do you prefer?

Reply:
I'm in favour of an API entry that states this, I don't feel what's-new is needed.

Reply:
✅ done.